database/mariadb/mariadb.def

```diff
@@ -2,10 +2,10 @@ Bootstrap: docker
 From: mariadb:10.3.9
 
 %post
-# <YOUR_USERNAME> is the user who will be executing the container,
-# just run: `whoami` and thats your username.
-# eg. sed -ie "s/^#user.*/user = westleyk/" /etc/mysql/my.cnf
-sed -ie "s/^#user.*/user = <YOUR_USERNAME>/" /etc/mysql/my.cnf
+# replace `your-name` with your username, run `whoami` to see your username
+YOUR_USERNAME="your-name"
+
```

Comment:
`YOUR_USERNAME` variable needs to be exported after its definition: `export YOUR_USERNAME`

Reply:
Same ... it's only used in that section so no need to export